lib/matplotlib/font_manager.py

```diff
@@ -1401,9 +1512,21 @@ def is_opentype_cff_font(filename):
 
 
 @lru_cache(64)
-def _get_font(filename, hinting_factor, *, _kerning_factor, thread_id):
-    return ft2font.FT2Font(
-        filename, hinting_factor, _kerning_factor=_kerning_factor)
+def _get_font(fpaths, hinting_factor, *, _kerning_factor, thread_id):
```

Comment:
I'm not sure of this.. we need fallback fonts while initializing a single FT2Font, so we either cache them all, or cache none of them..

If we cache all:
- can easily be invalidated if we have a different font family

If we cache one:
- we can't because we need the whole list of fallbacks for a single initialization

so it's better to cache all than to cache none?

Reply:
we need fallback fonts while initializing a single FT2Font, so we either cache them all, or cache none of them..

If we cache all:
- can easily be invalidated if we have a different font family

If we cache one:
- we can't because we need the whole list of fallbacks for a single initialization

so it's better to cache all than to cache none?